api/lib/notifications/notifications.js

```diff
@@ -212,7 +212,7 @@ const startNotificationService = (networks) => {
         topic,
         eventType: event.eventType,
         resourceType: event.resourceType,
-        resourceId: event.resourceId,
+        resourceId: String(event.resourceId),
```

Comment:
any reason not to `String()` everything here?

why does `resourceId` not being a string cause the whole row to be absent from the db?

Reply:
Yes, this looks definitely weird. In addition, when would the resourceId (an address) not be a String?

Reply:
the resource ID is the proposal ID which is an Int in Cosmos. this is not matching the GQL schema of our db. we can stringify everything for safety. go ahead

Reply:
shouldn't it be the proposer address? Ah no, alright, because it is a network related notification, I see.

OK, let's stringify everything then